renovate.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    ":automergeLinters",
+    ":automergeTypes",
+    ":ignoreModulesAndTests",
+    ":label(dependencies)",
+    ":pinVersions",
+    ":prConcurrentLimit10",
+    ":semanticCommitsDisabled",
+    "group:monorepos",
+    "group:recommended",
+    "npm:unpublishSafe",
+    "workarounds:all"
```

Comment:
I think this needs to be more specific, to prevent future (possibly unwanted) workarounds to just being blatantly accepted.

Reply:
Very well.  I left it there as it's part of their base config.  None of the workarounds are actually relevant to this repo.

I removed it in #15743.